pom.xml

```diff
@@ -164,6 +132,18 @@
             <version>${hamcrest.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>${mockito.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.spotify</groupId>
+            <artifactId>docker-client</artifactId>
+            <version>8.1.1</version>
```

Comment:
Can we move `version` to `properties` as well? The latest version is `8.9.2` can we use that?

Reply:
Updated